docs/_interface/dashboards.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Grafana Dashboard
+sort_key: S
+---
+
+There are several ways to monitor the performance of a Geth node. Insights 
+into a node's performance are useful for debugging, tuning and understanding 
+what is really happening when Geth is running.
+
+## Prerequisites {#prerequisites}
+
+To follow along with the instructions on this page you will need:
+
+- a running Geth instance.
+- basic working knowlegde of bash/terminal.
+
+[This video](https://www.youtube.com/watch?v=cOBab8IJMYI) provides an excellent 
+introduction to Geth monitoring.
+
+## Monitoring stack {#monitoring-stack}
+
+An Ethereum client collects lots of data which can be read in the form of a 
+chronological database. To make monitoring easier, this data can be fed into 
+data visualisation software. On this page, a Geth client will be configured 
+to push data into a InfluxDB database and Grafana will be used to visualize 
+the data.
+
+## Setting up InfluxDB {#setting-up-influxdb}
+
+InfluxDB can be downloaded from the [Influxdata release page](https://portal.influxdata.com/downloads/). 
+It can also be installed from a [repository](https://repos.influxdata.com/). 
+For example for a Debian based Linux operating system:
+
+```sh
+curl -tlsv1.3 --proto =https -sL https://repos.influxdata.com/influxdb.key | sudo apt-key add
```

Comment:
Very explicit instructions like these easily bitrot, might be better to link to the official download/install docs for influx?